### ISSUE TITLE
Questions about cell type

### ISSUE THREAD

**Issue by YiKangOY** [Entropy-xcy/sns#1]
Hello, I noticed that both in your code and your paper, only **mux** is considered as a cell type , while **pmux** will also appear after the yosys process to the verilog file.
Do you consider **pmux** as the same type as **mux**?

Thanks!

**Comment by Entropy-xcy** [comment]
Yes. Considering pmux as mux makes smaller vocab size and empirically increases the model accuracy.
